### DISCUSSION TITLE
OpenInv 4.3.0

### DISCUSSION THREAD

**Discussion by Jikoo** [Jikoo/OpenInv#131] [General]
## Supported Minecraft versions
1.18.2, 1.19.3, 1.19.4

## What's Changed
* Added support for 1.19.4
* Dropped support for 1.19/1.19.1
  * Policy is to support last 2 major revisions and last 2 minor revisions of current major revision
* Add lookup cache for `/openinv` and `/openender`
  * Saves last 10 lookup results
  * Improves performance for people who up-arrow every command no matter how long ago it was (I'm in this post and I don't like it)
* Add the option to have no arguments mean self for `/openinv` and `/openender`
  * Previously this always performed as the lookup cache by opening the last user searched
* Improve LanguageManager
  * Fix missing keys not being written to disk for easy translation
  * Add a guessfile-like system where missing translation keys are inserted into a separate section for easy identification
  * Reduce the verbosity of the warning for missing translations
* Clamp `/searchcontainer` radius to configurable amount
* Fix plugins causing redundant InventoryCloseEvents sticking players in an internal spectator state with SilentContainer
* Broke my own release process by trying to improve Actions usage
  * I had to manually download the binary to include it in the release which I am also manually creating. Very sad, at least 8 more clicks than usual. Automation in shambles. If I also broke the action that clones this release to Curseforge my day will be ruined.

**Full Changelog**: https://github.com/Jikoo/OpenInv/compare/4.2.2...4.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Jikoo/OpenInv/releases/tag/4.3.0'>OpenInv 4.3.0</a>.</em>

**Comment by PONZ69** [comment]
when will you add this to forge bukkit

**Reply by Jikoo** [reply]
If you're asking about dev.bukkit.org, the upload was automatically completed right after the release was created. It's up to their staff to actually approve the file.
